File: 백준/Bronze/2747. 피보나치 수/피보나치 수.cc
#include <iostream>

using namespace std;

int main() {
    int arr[46] = {0};
    int n;
    arr[0] = 0;
    arr[1] = 1;
    cin >> n;
    for(int i=2;i<=n;i++)
    {
        arr[i] = arr[i-2]+arr[i-1];
    }
    cout << arr[n];
}

File: 백준/Bronze/2747. 피보나치 수/피보나치 수.kt
fun main(args: Array<String>) {
    var n: Int = readLine() !!.toInt()
    var arr = Array(n+1,{0})
    arr[0] = 1
    arr[1] = 1

    for(i in 2 until n+1)
        arr[i] = arr[i-2]+arr[i-1]

    print(arr[n-1])
}